### PR TITLE
Human deathgasp + gasp sounds.

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Adds a sound effect for when humans gasp or die.

## Why It's Good For The Game

Lizards already have a deathgasp sound, and people often don't realize you're in crit until it's too late, this would make them notice faster.

## Changelog
:cl:
soundadd: added deathgasp and gasp sound for humans.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
this is cursed

**Comment by Effemu** [comment]
can you do them with /me?

**Comment by ghost** [comment]
> can you do them with /me?

You always were.

**Comment by BitBite112** [comment]
I dunno, I feel like they'll sound weird, but maybe it'll work out.

**Comment by qwertyquerty** [comment]
Shouldn't this be done in species code?

**Comment by ghost** [comment]
> Shouldn't this be done in species code?

This is where all the other audible emotes were.

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -82,6 +82,18 @@
 	message_simple =  "stops moving..."
 	stat_allowed = UNCONSCIOUS
 
+/datum/emote/living/deathgasp/get_sound(mob/living/user)
+	if(!ishuman(user))
```
Comment (ike709) [inline code/modules/mob/living/emote.dm:86]: Just put the type in the arg and `istype(user)` it

**Comment by ike709** [comment]
Post a video of the sounds so I don't have to download each of them.

**Review by PowerfulBacon — COMMENTED** [review]
Source for sounds?

**Comment by ghost** [comment]
> Source for sounds?

TGMC.

**Comment by ike709** [comment]
Sounds not posted after 6 weeks, marking it as stale.

**Comment by ghost** [comment]
> Sounds not posted after 6 weeks, marking it as stale.

Jesus Ike, I doubt that you don't have enough space on your computer to download a couple of OGG files, but here it is.

https://streamable.com/9qx6jx

**Comment by ike709** [comment]
Not bad, if a bit loud. What's the source for them?

**Comment by ghost** [comment]
> Not bad, if a bit loud. What's the source for them?

I took them from TGMC, I repurposed "human_male_gored 2" into the male deathgasp, and "human_female_gored 1" into the female deathgasp.

**Review by ike709 — DISMISSED** [review]

File: code/modules/mob/living/emote.dm [hunk 2]
```diff
@@ -82,6 +82,18 @@
...
+		return
+	var/mob/living/carbon/human/H = user
```
Comment (ike709) [inline code/modules/mob/living/emote.dm]:
```suggestion
/datum/emote/living/deathgasp/get_sound(mob/living/carbon/human/user)
	if(!istype(user))
		return
```

and you'll have to replace H with user

File: code/modules/mob/living/emote.dm [hunk 3]
```diff
@@ -150,6 +162,18 @@
 	emote_type = EMOTE_AUDIBLE
 	stat_allowed = UNCONSCIOUS
 
+/datum/emote/living/gasp/get_sound(mob/living/user)
```
Comment (ike709) [inline code/modules/mob/living/emote.dm:164]: ditto here

**Comment by Isy232** [comment]
I like this

**Comment by utilbird** [comment]
Failing checks

**Pull request closed without merging**